src/nodes/display/ToneMappingNode.js

```diff
@@ -114,7 +123,7 @@ const AGXToneMappingNode = Fn( ( { color, exposure } ) => {
 	colortone.assign( LINEAR_REC2020_TO_LINEAR_SRGB.mul( colortone ) );
 	colortone.assign( clamp( colortone, 0.0, 1.0 ) );
```

Comment:
Is the use of `assign()` the preferred style -- as is done in `AGXToneMappingNode`? It seems weird to me to `assign` to a `const` variable.

Reply:
Replacing variables should be recommended for assign like:

```js
color = color.mul( exposure );
color = color.sub( 0.004 ).max( 0.0 );
```

We have some extra `assign()` and `.toVar()` in some nodes because it's was transpiled, currently the TSL Transpiler does not implement this type of optimization.